gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/exception/AbstractSweException.java

```diff
@@ -37,6 +37,9 @@ public final String getDetails() {
             while (cause != null) {
                 sb.append("; nested exception is ").append(cause);
                 cause = cause.getCause();
+                if(cause instanceof SweInvalidDataNoDetailsException){
```

Comment:
The maven build does not work because it misses two spaces here: after the "'if' and before the "{"

Reply:
yes je suis en train d'essayer de voir pourquoi je n'arrive pas à lancer le checkstyle en local pour corriger